problems.sql

```diff
@@ -0,0 +1,78 @@
+-- Select titles from movies
+2.1
+SELECT title, id
```

Comment:
You could just use the `*` since you need to show everything about the selected movie. 
`SELECT * FROM movies WHERE id = 6`